subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,66 @@
+can be reached out to and engaged with regularly. A business model like this makes it easy to
+anticipate demand and supply. Finally, this model offers much higher payment security for your
+business.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
+   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
+   management.</b> </div>
+
+Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
+help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
+possibility to create, edit and manage your own subscription templates.
+
+Configuration
+=============
+
+Go to :menuselection:`Subscriptions → Configuration → Subscription templates`. By default, Odoo
+suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
+Of course, you can create your own ones.
+
+.. image:: media/subscription_templates_1.png
+  :align: center
+  :alt: How to use subscription templates on Odoo Subscriptions?
+
+.. important::
+   Installing the **Subscriptions** application will automatically install the **Sales** and
+   **Invoicing** applications.
+
+Create your first template
+==========================
+
+You can create a new template or edit an existing one. The first thing you need to do is give your
+template a name. After that, you have to choose an invoicing period. It is possible to change the
+amount and specify whether it is per days, weeks, months or years. The duration will determine if
+the subscription will go on forever (until it’s manually closed) or for a fixed amount of time.
+There are also several options for the payment mode. Something important here is that an additional
+field appears when you choose “Send”, “Send & try to charge” and “Send after successful payment”.
+This field gives you the possibility to add an invoice email template (as you can see below).
+
+.. image:: media/subscription_templates_2.png
+  :align: center
+  :alt: How to use subscription templates on Odoo Subscriptions?
+
+On each template, you can also configure some options for the customer portal. First of all, you are
+going to be able to say if you want the customer to be able to close the subscription or not. After
+that, you have the automatic closing limit, the group of subscription and the journal options that
+you can also manage directly from the subscription templates.
+
+.. note::
+   For each template, you can add your **Terms and Conditions** and your **Health Check**.
+
+.. important::
+   After creating your own subscription template, be sure to check out our documentation on how to
+   create, edit and manage your own subscription products: :doc:`subscription_products`, to complete
+   the sales flow.
+
+.. seealso::
+   - :doc:`subscription_products`
```

Comment:
As you added the seealso I would remove the paragraph. I believe the seealso is enough and it does the job of guiding the user through the flow (:

Reply:
I know this is repetitive but I like to put the "see also" if I mentioned a doc in the text :) It is two times but it is necessary for some customers ^^